verilog/parser/verilog.y

```diff
@@ -5479,6 +5479,18 @@ continuous_assign
     { $$ = MakeTaggedNode(N::kContinuousAssign, $1, $2, $3, $4, nullptr); }
   ;
 
+net_alias_assign
+  : lpvalue '=' net_alias_assign
+    { $$ = ExtendNode($1, $2, $3); }
+  | lpvalue
+    { $$ = move($1); }
+  ;
+
+net_alias
+  : TK_alias lpvalue '=' net_alias_assign ';'
```

Comment:
With the above list being at minimum a doubleton, this could be simplified to:
`: TK_alias net_alias_assign ';'`

Reply:
right!